webapp/pages/map.vue

```diff
@@ -257,7 +257,9 @@ export default {
           </div>
            `
         description +=
-          e.features[0].properties.about && e.features[0].properties.about.length > 0
+          e.features[0].properties.about &&
+          e.features[0].properties.about.length > 0 &&
+          e.features[0].properties.about !== 'null'
```

Comment:
How it commes that `e.features[0].properties.about` becomes a string with value 'null'.
That should not happen in my view.

So it should be this, but this is part of the first line and therefore superfluid:

```suggestion
          e.features[0].properties.about !== null
```